Commands/SitesMassWordPressUpdateCommand.php

```diff
@@ -471,4 +534,109 @@ private function send_to_slack($text) {
   private function isReport() {
     return isset($this->assoc_args['report']) ? true : false;
   }
+
+  private function isConfigFile() {
+    return isset($this->assoc_args['config-file']) ? true : false;
+  }
+
+  private function getSetting($name, $default_value = false) {
+    return isset($this->assoc_args[$name]) ? true : $default_value;
+  }
+
+  private function isExcluded($site) {
+    if ($this->isConfigFile()) {
+
+    } else {
+      $excluded= isset($this->assoc_args['exclude']) ? explode(',', $assoc_args['exclude']) : false;
+      
+    }
+    
+  }
+
+  private function parseYaml( $dir ){
+
+    if ( ! file_exists( $dir ) ) {
+
+      $this->log()->info( 'File ' . $dir . ' does not exists.' );
+
+      return false;
+
+    }
+
+    require_once 'lib/Spyc.php';
+
+    $yaml_settings = spyc_load_file( $dir );
+
+    $yaml_settings = $this->validateYamlSettings( $yaml_settings );
+
+    return $yaml_settings;
+
+  }
+
+  private function yamlGetExcludedSites( $yaml ){
+
```

Comment:
You could combine in this method the values in `$this->assoc_args['exclude'])` and return all the excluded sites. Doing that you could remvoe the `isExcluded` function. And then in the update method just ask `in_array(site, $excludedSites)`